docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+
+![Screenshot: Create a new article in Admin Panel](../assets/quick-start-guide/qsg-starters-part2-02-write_article.png)
+
+#### Give your article a title, a description, and some content
+
+1. Type `Hello World!` in the **Title** field.
+2. Type `My very first article with Strapi` in the **Description** field. 
+3. Write a few lines in the **Content** field. If you're lacking some inspiration, just type `This is my first blog article with Strapi and using it feels like a breeze!`.
+4. Scroll down, pick a date in the **PublishedAt** field.
+5. Finally, add a picture in the **Image** field. You can do this either by drag & drop, or by selecting a file from the Strapi assets library.
+
+#### Choose an author and a category for your article
+
+In the sidebar on the right, choose your name in the **Author** dropdown. 🥳 You have just signed your first article with Strapi. Take a few seconds to contemplate this historic moment!
+
+While there, you might also want to choose a **Category** for your article (or add your own) from the list.
+
+#### Turn your draft into a publication
+
+By default, your new article would be saved as a draft. Let's not be too shy and publish it right away:
+
+1. From the **Status** dropdown at the bottom of the page, choose **published**.
+2. Click on **Save** at the top of the window.
+
+![Animated GIF to create an article](../assets/quick-start-guide/qsg-starters-part2-03-write_publish_article.gif)
+
+You have just created and published your first article, `Hello World!`. You can find it in the [_Collection types > Articles_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::article.article?page=1&pageSize=10&_sort=id:DESC) view.
+
+### Step 3: Update the `Homepage` single type
+
+It's time to make this blog a bit more yours.
+
+Click on [_Single Types > Homepage_](http://localhost:1337/admin/plugins/content-manager/singleType/application::homepage.homepage) in the left-hand menu. Let's edit this homepage:
+
+1. Hover the picture in the **ShareImage field** and click the **Edit** icon <Fa-PencilAlt />.
+2. Click on **Replace Media** and upload an image from your computer. This image will represent your blog when sharing an article on social medias.
+3. Click **Finish** twice.
+4. At the bottom of the page, update the **Title** to `My Wonderful Strapi Blog` in the **Hero** field group.
+5. Finally, click **Save**.
+
+### Step 4: Restart the servers to reflect latest changes
+
+Gatsby is a static-site generator. It means that you need to restart the servers for changes to appear on the frontend:
+
+1. In your terminal, press `Ctrl-C` to stop the servers.
+2. Make sure you are in the `my-project` folder. If not, type `cd my-project` and press `Enter`.
+3. Restart the servers by typing `yarn develop` (or `npm run develop`) and press `Enter`.
```

Comment:
In the  **Part 2:** first **TIP** you used `npm` by default and `yarn` in parenthesis.
Here it's the opposite.
I would suggest having `npm` by default and yarn always in parenthesis.
Not everyone has yarn installed.

Reply:
Good catch, thanks! I tried to be consistent but forgot this one :D

Reply:
(But honestly, on a side note and IMHO, I really don't get why we don't advocate much more for `yarn` over `npm` in the company. Yarn is much, much faster.)